webpack.config.dev.js

```diff
@@ -16,7 +16,7 @@ module.exports = merge(common, {
   devServer: {
     contentBase: path.resolve(__dirname, "public"),
     port: 3005,
-    https: true,
```

Comment:
Did you mean to check this in?
Do we want the default to have https on or off?

Reply:
for dev purposes we can leave it off.